sdk/src/main/java/com/horizen/account/transaction/EthereumTransaction.java

```diff
@@ -186,19 +194,24 @@ public BigInteger getNonce() {
     public BigInteger getGasPrice() {
         if (!this.isEIP1559())
             return this.legacyTx().getGasPrice();
-        return null;
+        //in Geth for EIP1559 tx gasPrice returns gasFeeCap
+        return getMaxFeePerGas();
     }
 
     public BigInteger getMaxFeePerGas() {
```

Comment:
I think it depends if any account-based sidechain has the concept of gas and gas price. I suppose it may be but I'm not sure. @i-Alex what do you think?

Reply:
Done